test/train-sets/ref/topk-rec.stderr

```diff
@@ -12,20 +12,17 @@ loss     last          counter         weight    label  predict features
 0.000003 0.000003            1            1.0   3.0000   2.9983        4
 0.000004 0.000005            2            2.0   0.0000   0.0022        4
 0.000003 0.000000            3            3.0   2.0000   2.0000        4
-0.000003     n.a.            4            4.0  unknown   0.0000        1
-0.000003 0.000003            5            5.0   0.0000   0.0018        4
-0.000004 0.000011            6            6.0   3.0000   2.9968        4
-0.000004 0.000001            7            7.0   1.0000   1.0007        4
-0.000004     n.a.            8            8.0  unknown   0.0000        1
-0.000003 0.000000            9            9.0   2.0000   2.0004        4
-0.000003 0.000000           10           10.0   1.0000   1.0003        4
-0.000002 0.000000           11           11.0   3.0000   2.9995        4
-0.000002     n.a.           12           12.0  unknown   0.0000        1
+0.000003 0.000003            4            4.0   0.0000   0.0018        4
+0.000004 0.000011            5            5.0   3.0000   2.9968        4
+0.000004 0.000001            6            6.0   1.0000   1.0007        4
+0.000003 0.000000            7            7.0   2.0000   2.0004        4
+0.000003 0.000000            8            8.0   1.0000   1.0003        4
+0.000002 0.000000            9            9.0   3.0000   2.9995        4
 
 finished run
-number of examples = 12
-weighted example sum = 12.000000
+number of examples = 9
+weighted example sum = 9.000000
 weighted label sum = 15.000000
 average loss = 0.000002
 best constant = 1.666667
-total feature number = 39
+total feature number = 36
```

Comment:
This changes because you eliminate newline examples with a constant feature?

Reply:
Yes exactly